services/common/src/redux/customAxios.js

```diff
@@ -34,9 +93,22 @@ const CustomAxios = ({ errorToastMessage, suppressErrorNotification = false } =
         (errorToastMessage === "default" || errorToastMessage === undefined) &&
         !suppressErrorNotification
       ) {
+        console.error('Detailed Error: ', error?.response?.data?.detailed_error)
+        const notificationKey = 'errorNotification';
         notification.error({
+          key: notificationKey,
           message: formatErrorMessage(error?.response?.data?.message ?? String.ERROR),
+          description: <p style={{ color: 'grey' }}>If you think this is a system error please help us to improve by informing the system Admin</p>,
```

Comment:
Could we put this behind a feature flag in case we decide on doing any iterations on this before releasing?

Reply:
Added a feature flag from FE. Thanks